### PR TITLE
Release OTEL Operator 0.47.0

### PR DESCRIPTION
**Pull request by yuriolisa**
Signed-off-by: Yuri Sa <yurimsa@gmail.com>

### CODE REVIEW

**Comment by yuriolisa** [comment]
#774

**Comment by yuriolisa** [comment]
@jpkrohling , @VineethReddy02 , @pavolloffay . Could you please have a look?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,15 @@
 Changes by Version
 ==================
 
+0.47.0
+-------------------
+* Bumped OpenTelemetry Collector to v0.47.0
+* doc: customized auto-instrumentation ([#762](https://github.com/open-telemetry/opentelemetry-operator/pull/762), [@cuichenli](https://github.com/cuichenli))
+* Remove v prefix from the container image version/tag ([#771](https://github.com/open-telemetry/opentelemetry-operator/pull/771), [@pavolloffay](https://github.com/pavolloffay))
+#### OpenTelemetry Collector and Contrib
+* [OpenTelemetry Collector - v0.46.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.47.0)
```
Comment (pavolloffay) [inline CHANGELOG.md:10]: the version does not match

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+* [OpenTelemetry Contrib - v0.46.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.47.0)
```
Comment (pavolloffay) [inline CHANGELOG.md:11]: here as well

Comment (yuriolisa) [reply CHANGELOG.md]: Oh man, typos!!! =(

Comment (pavolloffay) [reply CHANGELOG.md]: :P

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**